api/document/templates/document/editor.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<style>
+#editor {
+  border: 1px solid black;
+}
+</style>
+<title>Document editor</title>
+<h1>Document editor</h1>
+<div id="editor"></div>
+{% load static %}
+<script src="{% static 'bundles/js/document.main.bundle.js' %}"></script>
```

Comment:
I know this is a placeholder, but would you mind adding `head`/`body` tags so we get started on the right foot?

Reply:
Oh sure thing. I've actually been starting with this ever since I found out it was actually valid HTML5, but yeah, I always end up having to add the other tags as the thing evolves, so might as well start with it!